projects/element-ng/datepicker/si-datepicker.component.ts

```diff
@@ -134,6 +135,27 @@ export class SiDatepickerComponent implements OnInit, OnChanges, AfterViewInit {
   readonly calenderWeekLabel = input<TranslatableString>(
     $localize`:@@SI_DATEPICKER.CALENDAR_WEEK_LABEL:Calendar week`
   );
+  /**
+   * @defaultValue
+   * ```
+   * $localize`:@@SI_DATEPICKER.CALENDAR_WEEK_LABEL:Calendar week`
+   * ```
+   */
+  readonly calendarWeekLabel = input<TranslatableString>(
+    $localize`:@@SI_DATEPICKER.CALENDAR_WEEK_LABEL:Calendar week`
+  );
+  private readonly defaultWeekLabel = $localize`:@@SI_DATEPICKER.CALENDAR_WEEK_LABEL:Calendar week`;
+  protected readonly derivedWeekLabel = computed(() => {
+    if (this.calendarWeekLabel() !== this.calenderWeekLabel()) {
+      if (this.calenderWeekLabel() !== this.defaultWeekLabel) {
+        return this.calenderWeekLabel();
+      } else {
+        return this.calendarWeekLabel();
+      }
+    } else {
+      return this.calendarWeekLabel();
+    }
```

Comment:
I think you can just remove the default value of `calendErWeekLabel` and then do `computed(() => calenderWeekLabel() ?? calendarWeekLabel())`